### ISSUE TITLE
Explicit License 

### ISSUE THREAD

**Issue by DennisHeimbigner** [vmxdev/tkvdb#11]
any chance of getting you to change to explicitly use and reference of of the standard licenses?
BSD 3 clause would be a good choice.
The reason I ask is that our code auditor for our project (unidata netcdf) uses a tool to ensure we only use a selected
set of well known licenses, so using an explicit URL to point to a standard license website keeps the tool happy.

**Comment by vmxdev** [comment]
Hello!

I'm not an expert in licenses, but it seems to me that ISC and BSD are almost the same. ISC is more permissive, that was the reason I chose it.
Can you fork and relicense under another license (maybe under another name)? For me it would be quite ok. If I need to do something for this - feel free to text me, I will do it.

**Comment by DennisHeimbigner** [comment]
Yes, if you could do that with some other name -- radixtrie perhaps -- using this URL as part of your license file: https://opensource.org/license/bsd-3-clause
The reason I ask this has to with the way our software (Unidata/netcdf-c) gets audited. They are picky about the  licenses for external software that we use.
BTW I am going by the ratings on the Open Source Initiative web site: https://opensource.org/license
They rate ISC as uncategorized (https://opensource.org/license/isc-license-txt).

**Comment by vmxdev** [comment]
No, I will not fork, rename and relicense the library.
I don't need it.
**You** can do it.
I meant that I can give special permissions if it is really necessary.
You can even take the library and remove any mention of authorship, I really don’t care.

**Comment by bogen85** [comment]
FSF lists it: https://www.gnu.org/licenses/license-list.html#ISC
"This license is sometimes also known as the OpenBSD License. It is a free software license, and compatible with the GNU"

[ISC License](https://directory.fsf.org/wiki/License:ISC) ([#ISC](https://www.gnu.org/licenses/license-list.html#ISC))
"Free and GPL-Compatible License. "

The tool that lists it as "uncategorized" should be updated...
(As should the Open Source Initiative web site...)
